### PR TITLE
meta tag info added

### PR DESCRIPTION
**Pull request by LaxmanJonchhen12**
meta tag information added for SEO optimization of the page and also added ALT Tag info for the SEO.

### CODE REVIEW

File: app/templates/public/index.html [hunk 1]
```diff
@@ -8,13 +8,21 @@
 {% block title %}College Predictor 2020{% endblock %}
 
 {% block main %}
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="
+  Welcome to College Predictor. College Predictor is a website which helps for IIT, JEE Aspirants to predict or find college and Branch you are likely to get into based on you percentile, quota, gender for JEE Mains 2020">
+  <meta name="keywords" content="IIT, JEE, College, Predictor, Engineering, 2020">
+  <meta name="author" content="Niharika Kabra">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ShubhamPalriwala) [inline app/templates/public/index.html:17]: Try to add some more meta stuff if possible !

**Review by ShubhamPalriwala — CHANGES_REQUESTED** [review]
Try to optimize it more !

File: app/templates/public/index.html (hunk 1 above)
Comment (LaxmanJonchhen12) [reply app/templates/public/index.html:17]: ok i have researched and added as much as i can still i would research and add

**Comment by LaxmanJonchhen12** [comment]
Hello I have bit researched and made change in meta description and added title to the link. I think this much is the meta information i can add for on page seo.

File: app/templates/public/index.html (hunk 1 above)
Comment (LaxmanJonchhen12) [reply app/templates/public/index.html]: Hello I have bit researched and made change in meta description and added title to the link. I think this much is the meta information i can add for on page seo.

Comment (LaxmanJonchhen12) [reply app/templates/public/index.html]: Hello I researched about on page seo and did much I can in project HTML page can you accept my pull request

**Comment by LaxmanJonchhen12** [comment]
> Try to optimize it more !

Hello I have done all the research on one page seo and i tried to optimize the page for good SEO as much as i can can your review and merge the pull request

**Pull request merged**